.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,42 @@
+{
+	"name": "devcontainer-fluent-bit",
+	"image": "fluent/fluent-bit:latest-debug",
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	"features": {
+		// add non-root user. it is similar your local user.
+		"ghcr.io/devcontainers/features/common-utils": {
+			"version": "latest",
+			"username": "fluent-bit",
+			"userUid": 1000,
```

Comment:
I think if we're mounting local files and changing them (or creating builds, etc.) we probably should match up the UID and GID of the user running it to avoid any weird file permission issues. 

Is there a good way to do this with devcontainers? Or does it handle it for you?
e.g. https://jtreminio.com/blog/running-docker-containers-as-current-host-user/

I'm not sure it's a blocker but it can cause weirdness locally if you're not aware of it.

Reply:
If we dont pass uid and gid, it will set automatically local user values